src/components/ParksList/ParksList.test.jsx

```diff
@@ -0,0 +1,1088 @@
+    activities: [
+      {
+        id: '1DFACD97-1B9C-4F5A-80F2-05593604799E',
+        name: 'Food',
+      },
+      {
+        id: '0B685688-3405-4E2A-ABBA-E3069492EC50',
+        name: 'Wildlife Watching',
+      },
+      {
+        id: '5A2C91D1-50EC-4B24-8BED-A2E11A1892DF',
+        name: 'Birdwatching',
+      },
+      {
+        id: '24380E3F-AD9D-4E38-BF13-C8EEB21893E7',
+        name: 'Shopping',
+      },
+      {
+        id: '467DC8B8-0B7D-436D-A026-80A22358F615',
+        name: 'Bookstore and Park Store',
+      },
+    ],
+    states: 'CA',
+    images: [
+      {
+        credit: 'NPS',
+        title: 'Alcatraz Island',
+        altText: 'View of the Alcatraz Lighthouse and Island from the water',
+        caption: 'Alcatraz Island',
+        url: 'https://www.nps.gov/common/uploads/structured_data/2514A14F-D5E3-BB31-4A0C4175BF61216A.jpg',
+      },
+      {
+        credit: 'NPS / Dave Rauenbuehler',
+        title: 'Alcatraz Cellhouse',
+        altText:
+          'A corridor extends between two rows and three tiers of cells. Skylights let in light from overhead.',
+        caption: 'Looking down Broadway in the Alcatraz Cellhouse',
+        url: 'https://www.nps.gov/common/uploads/structured_data/5482A294-DB42-56E0-FCCCD03C986AE1DC.jpg',
+      },
+      {
+        credit: 'NPS Photo',
+        title: 'Alcatraz Rangers and Firetruck',
+        altText: 'Alcatraz Rangers and Firetruck',
+        caption: 'Alcatraz Rangers and 1934 Diamond T Firetruck',
+        url: 'https://www.nps.gov/common/uploads/structured_data/A5C1D012-1DD8-B71B-0BA00C730D46F141.jpg',
+      },
+      {
+        credit: 'NPS Photo - Golden Gate National Recreation Area, GOGA 2316a',
+        title: 'Army Prisoners in the Stockade, 1902',
+        altText: 'Army Prisoners in the Stockade, 1902',
+        caption: 'Army Prisoners in the Stockade, 1902',
+        url: 'https://www.nps.gov/common/uploads/structured_data/A61F4F58-1DD8-B71B-0B981C552798242B.jpg',
+      },
+      {
+        credit: 'Darlyne Sheppard Alcatraz Photo Collection',
+        title: 'Serving the Christmas Meal, c 1951',
+        altText: 'Cook serving Christmas dinner with menu posted above.',
+        caption:
+          'An inmate worker distributes trays near the steam table. Bars separate the kitchen from the mess hall.  The day’s menu, the Christmas meal, appears on a sign over his head. The menu includes consomme, stuffed celery, green olives and mixed sweet pickles,',
+        url: 'https://www.nps.gov/common/uploads/structured_data/A64B4FF8-1DD8-B71B-0B63B232325C8081.jpg',
+      },
+    ],
+  },
+];
+
+test.only('should render a list of clickable parks', () => {
+  render(
+    <MemoryRouter>
+      <ParksList parks={parkResponse} />
```

Comment:
As discussed, a better test might be to render App and check that after clicking on the image the proper things are displayed